Patches/python.diff

```diff
@@ -75,62 +75,40 @@ diff -r 8667c26e2bec Include/dictobject.h
  #ifdef __cplusplus
 diff -r 8667c26e2bec Include/pystate.h
 --- a/Include/pystate.h	Mon Jun 15 09:11:37 2015 -0700
-+++ b/Include/pystate.h	Mon Aug 31 14:44:51 2015 -0700
-@@ -12,6 +12,8 @@
++++ b/Include/pystate.h	Thu Jan 07 11:46:53 2016 -0800
+@@ -12,6 +12,9 @@
  
  struct _ts; /* Forward */
  struct _is; /* Forward */
 +typedef void*(__stdcall *CompileFunction)(PyObject*);
++typedef void*(__stdcall *JitFreeFunction)(PyObject*);
 +typedef void(__stdcall *JitInitFunction)();
  
  #ifdef Py_LIMITED_API
  typedef struct _is PyInterpreterState;
-@@ -41,6 +43,7 @@
+@@ -41,6 +44,8 @@
  #endif
  
      PyObject *builtins_copy;
-+	CompileFunction jitcompile;
++    CompileFunction jitcompile;
++    JitFreeFunction jitfree;
  } PyInterpreterState;
  #endif
  
-diff -r 8667c26e2bec Lib/encodings/aliases.py
---- a/Lib/encodings/aliases.py	Mon Jun 15 09:11:37 2015 -0700
-+++ b/Lib/encodings/aliases.py	Mon Aug 31 14:44:51 2015 -0700
-@@ -1,4 +1,4 @@
--""" Encoding Aliases Support
-+﻿""" Encoding Aliases Support
+diff -r 8667c26e2bec Lib/test/test_list.py
+--- a/Lib/test/test_list.py	Mon Jun 15 09:11:37 2015 -0700
++++ b/Lib/test/test_list.py	Thu Jan 07 11:46:53 2016 -0800
+@@ -1,6 +1,7 @@
+ import sys
+ from test import support, list_tests
+ import pickle
++import unittest
  
-     This module is used by the encodings package search function to
-     map encodings names to module names.
-@@ -15,6 +15,7 @@
-         aliases have also been added.
- 
- """
-+
- aliases = {
- 
-     # Please keep this list sorted alphabetically by value !
-diff -r 8667c26e2bec Lib/importlib/_bootstrap.py
---- a/Lib/importlib/_bootstrap.py	Mon Jun 15 09:11:37 2015 -0700
-+++ b/Lib/importlib/_bootstrap.py	Mon Aug 31 14:44:51 2015 -0700
-@@ -1,4 +1,4 @@
--"""Core implementation of import.
-+﻿"""Core implementation of import.
- 
- This module is NOT meant to be directly imported! It has been designed such
- that it can be bootstrapped into Python as the implementation of import. As
-diff -r 8667c26e2bec Lib/importlib/_bootstrap_external.py
---- a/Lib/importlib/_bootstrap_external.py	Mon Jun 15 09:11:37 2015 -0700
-+++ b/Lib/importlib/_bootstrap_external.py	Mon Aug 31 14:44:51 2015 -0700
-@@ -1,4 +1,4 @@
--"""Core implementation of path-based import.
-+﻿"""Core implementation of path-based import.
- 
- This module is NOT meant to be directly imported! It has been designed such
- that it can be bootstrapped into Python as the implementation of import. As
+ class ListTest(list_tests.CommonTest):
```

Comment:
Weird. It's [upstream](https://hg.python.org/cpython/file/default/Lib/test/test_list.py#l4) so this may have been a temporary hiccup in the pegged version we use.